draft-poidt-ccamp-actn-poi-pluggable.md

```diff
@@ -147,42 +166,51 @@ informative:
    The full automation of the multilayer/multidomain network is a topic
    of high importance in the industry and the service providers
    community.  Typically, the layers composing such network are the IP/
-   MPLS, (with Segment Routing) and the Optical ones.  The requirements
+   MPLS packet domain (with Segment Routing) and the Optical domain 
+   providing DWDM transmission and photonic switching.  The requirements
    of high bandwidth availability and dynamic control of the networks
-   are of capital importance too.  The draft-ietf-teas-actn-poi-
-   applicability specifies very well how to control and manage
+   are of capital importance too.  
+   The {{!I-D.draft-ietf-teas-actn-poi-applicability}} 
+   specifies very well how to control and manage
    multilayer/multidomain networks using the Abstraction and Control of
    TE Networks (ACTN) architecture, see also {{my_figure-0}}.
 
-   New DWDM Coherent pluggable optics, such as ZR {{OIF-400ZR-01-0}} and
-   ZR+ {{Open_ZR-Plus_MSA}}, are enabling new multilayer network use cases
-   where the DWDM interface is located within the packet domain
-   equipment instead of being part of the Optical domain {{my_figure-1}}.
+   New pluggable 
+
+   Coherent pluggable digital coherent optics, such as ZR 
+   {{OIF-400ZR-01-0}} and ZR+ {{Open_ZR-Plus_MSA}}, are enabling new 
+   multilayer network use cases where the DWDM optical transmission
+   function is located within the packet domain equipment instead of 
+   being part of the Optical domain {{my_figure-1}}.
+   This means that an IP/MPLS capable device becomes a multi-technology
+   IP/MPLS/Optical device, as the optical connections (OTSi service
+   and media channels) start and end at such devices.
 
-   ZR and ZR+ (and also CFP2-DCO) deployment in routers has already
-   started and are expanding significantly.  The way the DWDM pluggable
+   The coherent pluggable interface deployment in routers has already
+   started and are expanding significantly.  The way the pluggables 
    are in general managed is not yet completely specified and defined by
    any standard and it is becoming an urgent matter to cover for Service
-   Providers.  Full end-to-end management solution of these DWDM
-   coherent pluggable optics, leveraging on ACTN hierarchical
+   Providers.  A full end-to-end management solution of these pluggable
```

Comment:
I suggest that this be reworded… "A full end-to-end management solution of these pluggable digital coherent optics is becoming critical to allow a wider deployment beyond simple point-to-point high-capacity link scenarios between two IP/ MPLS routers. It is proposed that this solution leverages and builds on the ACTN hierarchical architecture."

Reply:
OK. I think the paragraph is overstating the ACTN usage, but this is not a critical point.